### PR TITLE
Change default search method and results order used when searching for content on internal links

### PR DESCRIPTION
**Pull request by hvelarde**
Refs. #127

### CODE REVIEW

**Comment by rodfersou** [comment]
http://jenkins.plone.org/job/pull-request-4.3/175/

File: Products/TinyMCE/adapters/JSONSearch.py [hunk 1]
```diff
@@ -32,21 +32,14 @@ def getSearchResults(self, filter_portal_types, searchtext):
...
         if searchtext:
             plone_layout = self.context.restrictedTraverse('@@plone_layout',
                                                            None)
-            if plone_layout is None:
-                # Plone 3
-                plone_view = self.context.restrictedTraverse('@@plone')
-                getIcon = lambda brain: plone_view.getIcon(brain).html_tag()
-            else:
-                # Plone >= 4
-                getIcon = lambda brain: plone_layout.getIcon(brain)()
```
Comment (rodfersou) [inline Products/TinyMCE/adapters/JSONSearch.py]: why did you remove this?

Comment (hvelarde) [inline Products/TinyMCE/adapters/JSONSearch.py]: Plone 3?

**Comment by rodfersou** [comment]
Testing again http://jenkins.plone.org/job/pull-request-4.3/176/

**Comment by rodfersou** [comment]
@hvelarde @davisagli Green!

**Comment by hvelarde** [comment]
feedback please, @plone/ux-team

**Comment by tkimnguyen** [comment]
Sorry for the late comments, but you did ask :) Do we need an indication to the user that they're searching now only by title, or a way to change the sort order? (the relevance measure in the ZCatalog is a bit opaque, whereas title order is predictable).

**Comment by hvelarde** [comment]
@tkimnguyen I don't think so; current sort order is not usable on sites with a lot of content anyway. most users will not notice the change anyway.

BTW, we wanted to use the same settings we use in collective.cover as they have probe sane, but we left the sort order as relevance because @davisagli prefers that.

**Comment by hvelarde** [comment]
can we merge this @plone/framework-team?

**Comment by mauritsvanrees** [comment]
I see test failures: http://jenkins.plone.org/job/plone-4.3-python-2.7/1080/testReport/

**Comment by mauritsvanrees** [comment]
Fortunately they are locally visible to, on coredev 4.3.

**Comment by mauritsvanrees** [comment]
Ah, I've got it. I will soon push a fix.

Strange that Jenkins for the pull request did not catch this... It was clearly green.

**Comment by mauritsvanrees** [comment]
Done: 0361279301ca0e812e8f28f44dfb21264ede9aa5
